xmppserver/src/main/java/org/jivesoftware/openfire/disco/IQDiscoInfoHandler.java

```diff
@@ -230,19 +230,34 @@ public IQ handleIQ(IQ packet) {
      * @param node the node that the provider will handle.
      * @param provider the DiscoInfoProvider that will handle disco#info packets sent with the
      *        specified node.
+     * @return the provider that was previously associated to the node, if any.
```

Comment:
Changes to this file seem unrelated to the PR?

Reply:
Eek!

Reply:
should now be gone.